qualtran/bloqs/basic_gates/x_basis_test.py

```diff
@@ -119,3 +130,78 @@ def _keep_and(b):
     bloq = XGate().controlled(CtrlSpec(qdtypes=QUInt(n), cvs=1))
     _, sigma = bloq.call_graph(keep=_keep_and)
     assert sigma == {And(): n - 1, CNOT(): 1, And().adjoint(): n - 1, XGate(): 4 * (n - 1)}
+
+
+def test_meas_x_classical_sim() -> None:
+    bloq = MeasX()
+
+    with pytest.raises(ValueError, match='MeasX imparts a phase'):
+        _ = bloq.call_classically(q=0)
+
+    with pytest.raises(ValueError, match='Invalid classical value'):
+        _ = bloq.on_classical_vals(q=2)
+
+    results = [
+        do_phased_classical_simulation(bloq, {'q': 0}, rng=np.random.default_rng())
```

Comment:
what do you think about seeding our random generator in tests? usually we do; and theoretically line 151 or 152 could fail